src/lib/cache.ts

```diff
@@ -108,15 +98,18 @@ function _get<T>(key) {
         reject(err)
       } else if (data) {
         if (CACHE_COMPRESSION_DISABLED) {
-          resolve(JSON.parse(data.toString()))
+          resolve(JSON.parse(data.value.toString()))
```

Comment:
one API change, the returned data is an object with some metadata, `value` contains the actual cached object.